python/setup.py

```diff
@@ -102,10 +102,10 @@ def _supports_symlinks():
               file=sys.stderr)
         sys.exit(-1)
 
-# If you are changing the versions here, please also change ./python/pyspark/sql/utils.py and
-# ./python/run-tests.py. In case of Arrow, you should also check ./pom.xml.
+# If you are changing the versions here, please also change ./python/pyspark/sql/utils.py
+# For Arrow, you should also check ./pom.xml and ensure the Java version is binary compatible.
```

Comment:
can you expand on what ` and ensure the Java version is binary compatible` means?

Reply:
sure, will do